File: Shapes_Assighnment/Java files/Shapes.java
import java.util.Scanner;

class Shapes {
    public void getDimensions() {
        Scanner scanner = new Scanner(System.in); // Create a Scanner object to read user input
        System.out.println("Enter the length: ");
        length = scanner.nextInt(); // Read and store the length input by the user
        System.out.println("Enter the width: ");
        width = scanner.nextInt(); // Read and store the width input by the user
        scanner.close(); // Close the Scanner object to release resources
        compute(length, width);
    }

    public void compute(int length, int width) {
        int area, perimeter;
        area = length * width; // Compute the area of the shape
        perimeter = 2 * (length + width); // Compute the perimeter of the shape
        display(area, perimeter);
    }

    public void display(int area, int perimeter) {
        System.out.println("Area: " + area); // Display the calculated area
        System.out.println("Perimeter: " + perimeter); // Display the calculated perimeter
    }
}

File: Shapes_Assighnment/Python/Shapes.py
class Shapes:
    def getDimensions(self):
        length = int(input("Enter the length: "))  # Prompt user to enter length
        width = int(input("Enter the width: "))  # Prompt user to enter width
        self.compute(length, width)

    def compute(self, length, width):
        area = length * width  # Compute area by multiplying length and width
        perimeter = 2 * (length + width)  # Compute perimeter using the formula
        self.display(area, perimeter)

    def display(self, area, perimeter):
        print("Area:", area)  # Display the calculated area
        print("Perimeter:", perimeter)  # Display the calculated perimeter

if __name__ == "__main__":
    shapeObj = Shapes()  # Create an object of the Shapes class
    shapeObj.getDimensions()
